modules.d/90dm/dm-shutdown.sh

```diff
@@ -47,7 +47,7 @@ _do_dm_shutdown() {
 
 if command -v dmsetup > /dev/null \
     && [ "x$(dmsetup status)" != "xNo devices found" ]; then
-    _do_dm_shutdown $1
+    _do_dm_shutdown "${1-$final}"
```

Comment:
This changes behavior. Document in commit msg / Split out into separate commit?

Reply:
yeah, let me revert this .. it was made under false assumptions

Reply:
reverted